.github/workflows/ci-tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         # Something is wrong with windows tests, will need to debug separately.
         os: [ubuntu-20.04, macOS-11]
         # todo: seems there are some issues with py3.10 while install dependencies
-        python-version: ["3.7", "3.10"]
+        python-version: ["3.7", "3.9"]
```

Comment:
we still want to install 3.10, the issue is that 3.10 doesn't have all of the oldest things (such as torchvision)

Reply:
just fixed in [f624673](https://github.com/Lightning-AI/lightning-bolts/pull/930/commits/f624673bb9aa1010ddbfe3301746abfe83892034)

Reply:
in here :) 80be648